web/thesauruses/powershell/7/data_types.json

```diff
@@ -0,0 +1,110 @@
+{
+  "meta": {
+    "language": "powershell",
+    "language_name": "Human-Readable Language Name",
```

Comment:
> `powershell/7/data_types.json` has the default `language_name` attribute and needs to be updated

If you revise this, it should be good to merge as it will pass the checks.

Reply:
ok i will do this !